docs/current_docs/manuals/administrator/integrations/728101-tekton.mdx

```diff
@@ -0,0 +1,22 @@
+---
+slug: /manuals/administrator/728101/tekton
```

Comment:
@gerhard @kpenfound can you please review this page? I have based it on https://archive.docs.dagger.io/0.9/213240/tekton, modified to use a function call. However, I was not able to test it.